tensorboard/plugins/debugger/debugger_plugin_test.py

```diff
@@ -0,0 +1,436 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests the Tensorboard debugger data plugin."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import json
+import os
+import shutil
+import tempfile
+
+import mock
+import numpy as np
+import tensorflow as tf
+from werkzeug import test as werkzeug_test
+from werkzeug import wrappers
+
+# pylint: disable=ungrouped-imports, wrong-import-order
```

Comment:
There are some differences in the external and internal pylint checks. The internal one expects `import tensorflow as tf` and `from tensorflow.core.debug import debugger_event_metadata_pb2` to be in two separate groups, whereas the external one expect them to be together.